src/features/query-bar/__tests__/Condition-test.js

```diff
@@ -29,43 +29,69 @@ describe('features/query-bar/components/filter/Condition', () => {
         });
     });
 
-    describe('updateSelectedField()', () => {
+    describe('updateColumnField()', () => {
+        test('should select a column', () => {
+            const condition = initialCondition;
+            const columnId = '1';
+            const option = {
+                type: 'string',
+                value: columnId,
+            };
+            const onColumnChange = jest.fn();
+            const wrapper = getWrapper({
+                onColumnChange,
+            });
+
+            wrapper
+                .find('SingleSelectField')
+                .at(0)
+                .simulate('change', option);
+
+            expect(onColumnChange).toHaveBeenCalledWith(condition, columnId);
+        });
+    });
+
+    describe('updateOperatorField()', () => {
```

Comment:
Is this correctly named? I see that the tests below verify onFieldChange.

Reply:
Yes. updateOperatorField calls onFieldChange but the test itself is for testing that updateOperatorField correctly calls onFieldChange